.env.dev

```diff
@@ -5,7 +5,7 @@ REACT_APP_GUN_PUBLIC_URL=http://localhost:8765/gun
 REACT_APP_INFURA_KEY=
 REACT_APP_NETWORK=develop
 REACT_APP_WEB3_RPC=
-REACT_APP_WEB3_TRANSPORT_PROVIDER=
+REACT_APP_WEB3_TRANSPORT_PROVIDER=WebSocketProvider
```

Comment:
it is not regarding this task. Why u change this?

Reply:
@AlexeyKosinski I was searching for a reason of tests failing.

Reply:
Reverted .env.dev changes in a new commit